src/components/ArticleCard.jsx

```diff
@@ -31,8 +52,25 @@ export const ArticleCard = () => {
         <p className="author">Author: {newArticle.author}</p>
         <p className="dateMade">Written: {written}</p>
         <p>{newArticle.body}</p>
-        <img src={newArticle.article_img_url} />
-        <button className="votes">Vote: {newArticle.votes}</button>
+        <img className="img" src={newArticle.article_img_url} />
+        <div className="votesBox">
+          {voteError && <h4>{voteError}</h4>}
+          <button
+            onClick={() => handleClick(1)}
+            disabled={userVote > 0}
+            className={userVote > 0 && voteConfirm}
```

Comment:
Getting an error in the console when I vote because of a "className attribute returning a boolean" which I believe may be referring to this line here. Can you please fix this 🔧